components/searchBar/SearchBar.tsx

```diff
@@ -0,0 +1,38 @@
+'use client'
+
+import Image from 'next/image';
+import { useState } from 'react';
+
+import searchImg from '@/public/search.svg';
+import closeImg from '@/public/close.svg';
+
+import './SearchBar.css';
+
+const SearchBar = () => {
+  const [keyword, setKeyword] = useState('');
+  const handleKeywordChange = (e: any) => setKeyword(e.target.value);
```

Comment:
```suggestion
  const handleKeywordChange: ChangeEventHandler<HTMLInputElement> = (e) => setKeyword(e.target.value);

```

이렇게 하면 이벤트 핸들러 타입을 쉽게 관리할 수 있습니다.